spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Restaurant, type: :model do
+  subject { described_class.new(name: 'Burgerim', cuisine: 'Burger', address: 'some st. 22 Tel-Aviv')}
```

Comment:
I see you added the shoulda gem. You can test the validations like so:
```
describe Restaurant do
# Model member verification
 it { is_expected.to validate_presence_of :name }
 it { is_expected.to validate_presence_of :cuisine }
 it { is_expected.to validate_presence_of :address }
 it { is_expected.to validate_presence_of :max_delivery_time }
 it { should have_many(:reviews) }
etc.
```

Reply:
👍